stat/distuv/triangle.go

```diff
@@ -162,6 +175,9 @@ func (t *Triangle) UnmarshalParameters(p []Parameter) {
 		panic("triangle: " + panicNameMismatch)
 	}
 
+	// verify parameters are valid before directly setting
+	NewTriangle(a, b, c, src)
```

Comment:
`a`, `b` and `c` don't exist, but please don't do it this way; factor the check out to a helper, `checkTriangleParameters` and call it from both places. Then the comment can go away (note that we enforce capitalisation and punctuation on internal comments also).